Update components view with OntologyBrowserPanel

Sigmonia opened the pull request:
#### Rationale
Add the new OntologyBrowser component to the UI components view.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/1942
* https://github.com/LabKey/labkey-ui-components/pull/439
* https://github.com/LabKey/ontology/pull/21

#### Changes
* Update core-components.view with the new OntologyBrowserPanel
* Bumped core/package.json to latest w/ alpha

Sigmonia committed:
Add OntologyBrowserPanel to components view

Sigmonia committed:
update to components alpha package .0

Sigmonia committed:
missed package-lock

cnathe commented on core/src/client/LabKeyUIComponentsPage/LabKeyUIComponentsPage.tsx:
```diff
@@ -364,6 +366,11 @@ export class App extends React.Component<any, State> {
                 }
+                {selected === 'OntologyBrowser' &&
+                    this.renderPanel('OntologyBrowser',
+                        <OntologyBrowserPanel ontologyId={ 7 } />
```
if we end up leaving this on the components page, we'll want to have a way for the user to set the ontologyId for this param, but we can figure that part out later (especially since we might end up moving this panel to the ontology view/page).

Sigmonia committed:
Serialization and field updates for the ontology concept interfaces

Sigmonia committed:
*In-progress* Making the tree load
- Updating some of the OntologyController Actions to make it easier to load client side
- Updated storybook items

TODO:
Style clean up
Load/Cache concepts
Display concept info

Sigmonia committed:
*in-progress* link node selection through to info panel

TODO:
fix perf.
Clean up commented code.

cnathe committed:
Merge branch 'develop' into fb_ontBrowser

# Conflicts:
#	core/package-lock.json
#	core/package.json

cnathe committed:
Update to latest @labkey/components alpha package version

Sigmonia committed:
Add index to hierarchy.parent_code
Get concept on node selection
* pre-loading all child concepts can be expensive if we aren't looking at very many.
  Individually, they should load fairly quickly...

Sigmonia committed:
bump alpha version of components library

cnathe committed:
Add in an option for core-components OntologyBrowserPage.tsx for user to enter in the ontology id/name

cnathe committed:
Hide ontology input after set

cnathe committed:
Update to latest @labkey/components alpha package version

cnathe committed:
Update to latest @labkey/components alpha package version

cnathe committed:
Update to latest @labkey/components alpha package version

Sigmonia committed:
Update core package.json with @labkey/components with release version

Sigmonia committed:
merge develop into fb

Sigmonia committed:
CR Feedback
* Adjust ChildPaths query to return flag for hasChildren
 * Children are a path's property not Concept property
 * Can be slow for large sets
 * removed flag from Concept queries
* Removed some unused queries
* Added hasChildren and Label properties to ConceptPath interface
 * fields are needed for the client-side display
* Adjusted getConcept for Ontology root

Sigmonia committed:
merge develop into fb
